CHANGELOG.md

```diff
@@ -6,6 +6,7 @@ Upcoming
 
 - Drop support for Python2.7
 - Require `beets>=1.4.7`
+- Fix a crash in `beet check --add` when a music file is not found on disk.
```

Comment:
Feel free to add your github handle in parentheses ;)

Reply:
OK,  done